kateryna/databases.py

```diff
@@ -144,8 +144,8 @@ def close(self):
 
 
 sql = SQLiteApp(any_attr="some_value", connection=Connection(db_name="My Db"))
-print(sql.connect()) # connection to MyDb
-print(sql.execute("SELECT")) # executed
+print(sql.connect())  # connection to MyDb
```

Comment:
You've approved changes instead of requesting them, so I was not going to correct 😉